adaptive_navigation/pubspec.yaml

```diff
@@ -4,12 +4,14 @@ version: 0.0.1
 homepage: https://github.com/material-components/material-components-flutter-adaptive
 
 environment:
-  sdk: ">=2.7.0 <3.0.0"
+  sdk: ">=2.10.0-0.0.dev <3.0.0"
 
 dependencies:
   flutter:
     sdk: flutter
-  adaptive_breakpoints: ^0.0.2
+  adaptive_breakpoints:
+    # TODO: Return to normal.
```

Comment:
once reviews are complete, you should be able to publish the 3 packages in order, and then avoid these TODOs

Reply:
As of now, since NNBD is not released as beta yet, we cannot publish new packages to pub.dev. Once NNBD beta is released, we will be able to publish them in order and change to pub dependencies.
I updated the TODO.

Reply:
oh I thought you could still publish new ones as pre-releases, is that not the case?

Reply:
Pre-releases are only allowed for some crucial packages (that have a lot of downstream packages). It is recommended that other packages use a separate NNBD branch until NNBD beta is released.